<?xml version="1.0" ?>
<launch>
	<arg name="arm_prefix" value="sia20d_arm"/>
	<group ns="$(arg arm_prefix)">
		<node name="mantis_sia20d_sphere_pick_place" pkg="mantis_object_manipulation" 
			type="mantis_sphere_picking_node" output="screen">		
			<rosparam command="load" file="$(find mantis_sia20d_arm_navigation)/config/mantis_sphere_picking_parameters.yaml"/>
		</node>  
	</group>
</launch>
